internal/pkg/cli/env_delete.go

```diff
@@ -186,6 +186,10 @@ func (o *deleteEnvOpts) askEnvName() error {
 	for _, env := range envs {
 		names = append(names, env.Name)
 	}
+	if len(names) == 0 {
+		return fmt.Errorf("no environment found: run %s first please", color.HighlightCode("env init"))
```

Comment:
Recommending to run `env init` in the context of having run `env delete` feels strange to me.

Reply:
addressed